example/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
+    "android": "react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res && react-native run-android",
     "compile": "tsc -p .",
-    "ios": "react-native run-ios",
+    "ios": "react-native run-ios --simulator='iPhone 14'",
```

Comment:
Any reason to change this?

Reply:
@akshaybhalotia still react-native not able to run in iphone 14 with default "npx react-native run-ios" command. its taking iphone 13 simulator by default in latest Xcode iphone 13 simulator not present . so we have added iphone14 in command , with this command it will run in iphone simuator .

Reply:
Let this be added by the dev if needed, we don't need to enforce the simulator version at plugin level